src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,22 @@
 import React from 'react';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+
+import './CommentList.scss';
+
+import { Comment } from '../../types/Comment';
```

Comment:
```suggestion
import './CommentList.scss';
import { Comment } from '../../types/Comment';
import { CommentInfo } from '../CommentInfo';
```
No need to provide newlines between imports